server-spi-private/src/main/java/org/keycloak/models/BrowserSecurityHeaders.java

```diff
@@ -94,4 +94,52 @@
         defaultHeaders = Collections.unmodifiableMap(dh);
         headerAttributeMap = Collections.unmodifiableMap(headerMap);
     }
+
+    public static class ContentSecurityPolicyBuilder {
+
+        private String frameSrc = "self";
+        private String frameAncestors = "self";
+        private String objectSrc = "none";
+
+        private boolean first;
+        private StringBuilder sb;
+
+        public static ContentSecurityPolicyBuilder create() {
```

Comment:
Yes, I added it initially, but @pedroigor mentioned that it probably isn't something folks will want to set, so removed it. We are planning on introducing stricter CSP header soon and I imagine this will need updating then, also if someone wants it, it's just a quick PR away ;)

Reply:
Ah Ok, Seeing Pedro's comment now :) Thanks for explain